backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/impexp/format/ImpFormat.java

```diff
@@ -42,7 +47,9 @@ private ImpFormat(
 			final boolean multiLine,
 			final boolean manualImport,
 			@NonNull final ImportTableDescriptor importTableDescriptor,
-			@NonNull @Singular final List<ImpFormatColumn> columns)
+			@NonNull @Singular final List<ImpFormatColumn> columns,
+			@NonNull final String charset,
```

Comment:
consider java.nio.charset.Charset (or some other immutable alternative) instead of String

Reply:
Done